Kitodo/src/main/java/de/sub/goobi/beans/Benutzer.java

```diff
@@ -176,12 +175,18 @@ public void setPasswort(String inpasswort) {
 		this.passwort = inpasswort;
 	}
 
+	/**
+	 * @return add description
```

Comment:
I understand - a task list of that size is not useful. But you could add one TODO comment per file, something like `TODO: fix all "add description" comments`. Or you write an issue on GitHub, so maybe some volunteers see where they can help. All that is not necessary of course if you already have plans to fix the comments.